### ISSUE TITLE
Photo gallery editing

### ISSUE THREAD

**Issue by DivanX10** [t0mer/deepstack-trainer#8]
Thank you for adding the gallery, it became just super. Tell me, is there a possibility, and is it even possible to add a photo deletion function to the gallery and edit the name of the photo?

**Comment by t0mer** [comment]
Hi,
Everything is possible (:
I can add Photo Deletion and even renaming.
Just to clarify, the renaming will not affect the DeepStak AI database, only the display name in the gallery.
Will add it in the upcoming version.

**Comment by DivanX10** [comment]
Tell me, is the project abandoned or alive?  Will there be a photo editing function in the gallery?  You can also, for example, expand the functionality, add photos via telegrams, is it possible to do this?

**Comment by t0mer** [comment]
Hi, the project is still alive.
I had a lot going on lately so it took me some time to get back working on it.
I have added the ability to edit the gallery, hope to release it by the end of this week.
as for the telegram, yes it's an option, but it will take time (Working on some other projects in parallel).

**Comment by DivanX10** [comment]
Thank you for responding. I am very glad that your project is alive and will be developed. For my part, I introduce people to deepstack and show its capabilities. People are starting to show interest in deep stack. Links to my video. The video is in Russian, but you can include subtitles with translation into your native language

https://youtu.be/8ZyxROG9hgU
https://youtu.be/74b2bQDn1xA

**Comment by t0mer** [comment]
Wow, Thanks!
As for the gallery edit, I just released it...
* To change the image name, click on the name, it's editable end after renaming click the save button.
* To Delete image, just click the Delete button and that it.

About your telegram request, what do you expect? Just to add to the gallery or actually teach the system a new face?

Cheers!

**Comment by DivanX10** [comment]
Wow, it's already done. I'll try out the new image and unsubscribe later. 😀

> About your telegram request, what do you expect? Just to add to the gallery or actually teach the system a new face?

The main thing is that there is an opportunity, and you can always figure out how to use it. For example, I am not at home, but a friend or someone else should come to my house, whom I plan to let into the house and so that the recognition system does not raise the alarm when security is turned on and does not identify as a stranger, you need to inform her that access is allowed for this person. by sending a photo via telegram, this photo will be uploaded to the database and the system will recognize the guest as its own person and will not sound the alarm.

If possible, if it can be done at all. add and delete photos using telegram or rename photos

**Comment by DivanX10** [comment]
Thank you for the added feature. It works perfectly. 😀 🤟
